tailscale/DOCS.md

```diff
@@ -311,6 +312,17 @@ When not set, this option is enabled by default.
 
 Received files are stored in the `/share/taildrop` directory.
 
+### Option: `udp_port`
+
+UDP port to listen on for WireGuard and peer-to-peer traffic.
+
+Use this option (and router port forwarding) if you experience that Tailscale
+can't establish peer-to-peer connections to some of your devices (usually behind
+CGNAT networks). You can test connections with `tailscale ping
+<hostname-or-ip>`.
```

Comment:
Should we use the network ports natively supported by add-ons instead?

Reply:
Yeah, you are right, this port is available on the local interfaces, not only on the tailnet, like the proxy_and_funnel_port.

The only strange user experience is, that it is hidden behind a "Show disabled ports" button, meanwhile it is not disabled but randomly selected.